### ISSUE TITLE
Bedwars arena saving error

### ISSUE THREAD

**Issue by Duckhunterone** [MysticCity/MinigamesAPI#323]
Hi i hope you can help.
I have already set up one arena and it saved and everything perfect but when i try to do the same for my second arena i keep getting an error. its probably something im doing wrong but any help you can give will be appreciated  http://prntscr.com/gjwp0j

**Comment by mepeisen** [comment]
Did you set the spawns correctly for the arena?

**Comment by Duckhunterone** [comment]
yeah it seems i did but ive only just started working on the plugin so it may be a fault on my end. im gonna work on it over the next week or so and see how i get on and if im still having problems ill get in touch.
thanks for your quick reply.
James
AKA Duckhunterone

**Comment by Purplegigabyte** [comment]
yeah I'm having the same issue can anyone help plz : )

**Comment by ConnorFranta** [comment]
From the debug message, it's saying you need the player spawn points set and possibly your low and high arena points.

How to set up BedWars:

1. /bw setmainlobby - sets the main lobby where all the join signs will be
2. /bw setlobby [arena] - sets the arena waiting lobby
     (in a diagonal cube set the lobby bounds below)
  - /bw setlobbybounds [arena] low - sets the waiting lobby bounds
  - /bw setlobbybounds [arena] high - sets the waiting lobby bounds
3. Create the arena bounds (in a diagonal cube)
  - /bw setbounds [arena] low - sets the arena bounds
  - /bw setbounds [arena] high - sets the arena bounds
4. /bw setspawn [arena] - sets the player spawn, repeat for multiple spawns
5. Set min and max players for the arena
  - /bw setminplayers [arena] <count>
  - /bw setmaxplayers [arena] <count>
6. /bw setupbeds [arena] - you will get your beds in your inventory, one for each team color. Place them at the home base of the teams. After placing them you can remove them from inventory.
7. /bw savearena [arena] - creates a save of the arena

Find more info  on setup, go here - https://github.com/MysticCity/MinigamesAPI/wiki/Admin-Guide
Find the commands here, just add /bw before them - https://github.com/MysticCity/MinigamesAPI/wiki/Default-Sub-Commands
